pkg/api/login_test.go

```diff
@@ -687,6 +687,7 @@ func TestLoginPostSendLoginLog(t *testing.T) {
 
 			cmd := testReceiver.cmd
 			assert.Equal(t, c.cmd.LogAction, cmd.LogAction)
+			assert.Equal(t, c.cmd.LoginUsername, "admin")
```

Comment:
I think you want to do the following? The second argument to assert.Equal is the expectation, and the third is what you got. What you got should be cmd.LoginUsername right?
```suggestion
			assert.Equal(t, "admin", cmd.LoginUsername)
```

Reply:
Yes exactly, I just fixed it, thanks! I'm still not used to put the expected value first.